src/Microsoft.AspNet.Mvc.Core/ContentResult.cs

```diff
@@ -27,9 +27,9 @@ public class ContentResult : ActionResult
         public string Content { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="MediaTypeHeaderValue"/> representing the Content-Type header of the response.
+        /// Gets or sets the media type representing the Content-Type header of the response.
         /// </summary>
```

Comment:
Yeah let's fix all of these. `Gets or sets the Content-Type header of the response.`